frontend/src/pages/gov/Proposals.js

```diff
@@ -46,13 +48,38 @@ export default function Proposals({ chainUrl, chainName, chainLogo }) {
   const chainInfo = useSelector((state) => state.wallet.chainInfo);
   const walletConnected = useSelector((state) => state.wallet.connected);
   const [proposals, setProposals] = useState([]);
+  const [authzEnabled, setAuthzEnabled] = useState(false);
   useEffect(() => {
     if (walletConnected) {
       const response = async (chainUrl) => {
         const res = await govService.proposals(chainUrl);
         return res.data;
       };
-      response(chainUrl).then((res) => {setProposals(res.proposals)});
+      response(chainUrl).then((res) => {
+        setProposals(res.proposals);
+      });
+
+      const grantsToMeResponse = async (data) => {
```

Comment:
we have to use try and catch block whenever we use async-await.
```suggestion
      const grantsToMeResponse = async (data) => {
           try {
              const response = await service.grantsToMe(
                      data.baseURL,
                      data.grantee,
                      data.pagination
                    );
                    return response.data;
           } catch (e){
             throw e
           }
      }
```